### DISCUSSION TITLE
[Help] I don't have access, but I did have earlier (Visual Studio)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#14044] [Copilot Conversations] [answered]
So I installed the copilot extension in visual studio on my PC before I got access, and when I got the email confirming that I had access I installed it on my laptop as I was out of the house.
It worked perfectly fine, until I came back home and went to use it on my PC.
It says in the output window that I don't have access, I tried reinstalling the extension, relogging in visual studio, and even revoking the app's access to my account, and nothing seems to work, except now I get the same issue on my laptop too.
I'm all out of ideas on how to try and fix this, any help would be appreciated!

I'm running with Visual Studio 2022 version 17.1.3 Community Edition
Copilot 1.10.0.3
Agent 1.10.0

UPDATE: I deleted the copilot data in the visual studio registry, and when it loaded back in copilot asked me if I wanted to join the waitlist.
Seems like there is an issue with it checking if you've already gained access.

UPDATE 2: Reinstalled Visual Studio. Still no access. Now I need to set all the settings again.

**Comment by andre3s** [comment]
Well, I received the invitation and installed it on my PyCharm. After I authorized my laptop a message pop up appeared saying that I don't access to CoPilot and I should join the waiting list. I don't understand what happened.

**Comment by AngeloDotNet** [comment]
I also have the same problem with Visual Studio, despite having received the email to be able to use GitHub Copilot, opening any project (new or previously created), when loading the plugin I am shown the message "You don't have access to GitHub Copilot"

- Visual Studio 2022 17.1.3 Community Edition
- GitHub Copilot 1.10.0.3 v1.10.0.3
- GitHub Copilot Agent v1.10.0

**Comment by alysidev** [comment]
I have the same issue. Tried reinstalling the extension multiple times. No popup window shows.

**Reply by ghost** [reply]
You can make it re-show the "join waitlist" popup by deleting the copilot settings in the visual studio registry, but I'm afraid that it doesn't really achieve anything. It'll just take you back to the webpage to join the waitlist where it says that you already have access.

**Reply by yuba30** [reply]
@PersonMedBrukernavn if you get a chance can you let us know where in the registry you deleted the settings? Thanks!

**Reply by ghost** [reply]
@yuba30 You need to open regedit, click on HKLM, then go into File -> Load Structure
Then go to %localappdata%\Microsoft\VisualStudio\\[VS Version]\ and select privateregistry.bin and name it whatever
After loading in the registry hive go to HKLM\\[Name]\Software\Microsoft\VisualStudio\\[VS Version]\ApplicationPrivateSettings\GitHub\Copilot
Once you are finished remember to click on [Name] and go to File -> Unload Structure, or you might get some issues running VS

**Reply by yuba30** [reply]
Thanks!

**Reply by alysidev** [reply]
I found a separate solution late last night, but thanks!

**Comment by AngeloDotNet** [comment]
I've tried resetting Visual Studio settings, reinstalling Visual Studio, reinstalling the GitHub Copilot plugin, but the message "You don't have access to GitHub Copilot."

When I try to login from the plugin, it sends me back to the Copilot White List page, but trying to register again I am shown the message that for my github profile, copilot is already active.

Has anyone managed to fix the problem ?

**Comment by alysidev** [comment, marked as answer]
I figured out a solution! Go into your appdata local folder, delete github-copilot folder while the extension is still installed. go back and reload visual studio, reinstall the plugin, and the prompt for authorization will come up again after it is reinstalled! :)

**Reply by ghost** [reply]
Hey thanks for the tip, it works now and I didn't even have to reinstall the plugin!

**Reply by AngeloDotNet** [reply]
Thanks @alysidev for the solution, I also managed to solve the problem with Copilot
